package.json

```diff
@@ -47,13 +47,16 @@
     "@types/bcryptjs": "2.4.2",
     "@types/bluebird": "3.5.33",
     "@types/connect-flash": "0.0.36",
+    "@types/connect-history-api-fallback": "1.3.3",
     "@types/cors": "2.8.9",
+    "@types/cypress": "^1.1.3",
```

Comment:
@diminutivesloop Thanks for your contribution! Would you pin this version to 1.1.3. (remove the ^)